sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserRoleService.java

```diff
@@ -77,6 +78,18 @@ public UserRole getByCaption(String caption) {
 		return entity;
 	}
 
+	public UserRole getByLinkedDefaultUserRole(DefaultUserRole linkedDefaultUserRole) {
```

Comment:
Question: What happens if multiple user roles are linked to the same default user role? Do we prevent this from happening? Do we WANT to prevent this from happening? The logic currently doesn't seem to support something like that, but in practice I can absolutely see admins creating user roles that are very similar to another, with only minor differences, and linking them all to the same default user role.

Reply:
good point
since the method is only used in the startup service the current implementation findFirst should solve it
but i added an order on the creation date just to be sure and covered it with a test